dataactbroker/handlers/fileHandler.py

```diff
@@ -648,3 +722,10 @@ def completeGeneration(self, generationId):
         if generationId is None:
             return JsonResponse.error(ResponseException("Must include a generation ID",StatusCode.CLIENT_ERROR), StatusCode.CLIENT_ERROR)
 
+        # Pull url from request
+        safeDictionary = RequestDictionary(self.request)
+        url =  safeDictionary.getValue("href")
+        #Pull information based on task key
+        task = self.interfaces.jobDb.session.query(FileGenerationTask).options(joinedload(FileGenerationTask.file_type)).filter(FileGenerationTask.generation_task_key == generationId).one()
```

Comment:
Curious if there's a specific reason for doing an eager load here when joining these tables (as opposed to specifying a relationship in the model like we do for other foreign keys)?